src/v0/destinations/braze/README.md

```diff
@@ -0,0 +1,240 @@
+  - When merging users, be aware that some data might not be merged if the merge_behavior is set to "none"
+
+### Proxy Delivery
+
+- **Supported**: Yes
+- **Source Code Path**: `src/v0/destinations/braze/networkHandler.js`
+- Handles both v0 and v1 proxy requests
+
+### User Deletion
+
+- **Supported**: Yes
+- **Source Code Path**: `src/v0/destinations/braze/deleteUsers.js`
+- Implements the Braze User Delete API to comply with privacy regulations
+
+### Additional Functionalities
+
+#### Deduplication
+
+- **Supported**: Yes
+- **Configuration**: Enable via `supportDedup` setting
+- **How It Works**:
+
+  - When enabled, RudderStack maintains a user store with previously sent attributes
+  - For each user, attributes that have already been sent are removed from subsequent requests
+  - If all attributes for a user have been sent before, the user is dropped from the request
+  - The deduplication process uses Braze's `/users/export/ids` endpoint to fetch current user profiles
+  - RudderStack then compares incoming attributes with what's already in Braze
+
+- **Important Limitations**:
+
+  - This is a RudderStack-specific feature, not a Braze feature. Braze itself does not deduplicate events
+  - Deduplication only affects user attributes, not events or purchases
+  - Events and purchases will always be sent to Braze, even if they appear to be duplicates
+  - This can lead to duplicate events during data replay scenarios
+
+- **Use Cases**:
+
+  - Reducing data point consumption in Braze by preventing redundant attribute updates
+  - Preventing duplicate user profiles during data replay of Identify events
+  - Optimizing API usage by only sending changed attributes
+
+- **Implementation Details**:
+  - For Track events with user attributes, those attributes are deduplicated
+  - For Identify events, if all attributes have been sent before, the user is dropped from the request
+  - The deduplication logic is implemented in the `BrazeDedupUtility` class and `processDeduplication` function
+
+#### User Attributes
+
+- Supports comprehensive user profile management
+
+#### Event Tracking
+
+- Supports custom event tracking with properties
+
+#### Purchase Event
+
+- Special handling for purchase/e-commerce events
+- Standard properties: product_id, sku, price, quantity, currency
+
+#### Nested Array Operations
+
+- Supports operations on nested arrays in user attributes
+
+#### Nested Object Operations
+
+- Supports operations on nested objects in user attributes
+
+#### Reserved Properties Handling
+
+- Handles reserved properties according to Braze specifications
+
+#### Custom Attribute Operations
+
+- Supports various operations on custom attributes:
+  - Remove
+  - Update
+  - Add
+  - Create
+
+## General Queries
+
+### Event Ordering
+
+- **Required**: No specific ordering requirements documented
+- Braze processes events based on the timestamp provided
+
+### Data Replay Feasibility
+
+#### Missing Data Replay
+
```

Comment:
General rule for identify events: replay should not be recommended as there are high chances of user profile getting updated with stale data

Reply:
Also segregate replay feasibility per message type